### PR TITLE
Test query add user

### PR DESCRIPTION
**Pull request by ahmedalmi95**
test for add user query for first name and last name and mobile number.

### CODE REVIEW

**Review by Fatmasiam — APPROVED** [review]

**Review by Angham116 — APPROVED** [review]

File: src/database/queries/addData.js
Comment (tomduggan85) [inline src/database/queries/addData.js:4]: @AhmedAl-Almi Looks good!  You could use `hashPass` to hash the password right in this function, once #72 is merged.  I see that `hashPass` returns a promise so this function then will have to change slightly, to wait for the promise.  This looks great for now!

**Review by Fatmasiam — CHANGES_REQUESTED** [review]
CONFLICT !!!!

**Comment by kannanhassouna** [comment]
Closed Pull Request,
### DON'T MERGE THIS PULL REQUEST

**Pull request closed without merging**
